bitmark-references/bitmark-node-software/bitmark-blockain.md

```diff
@@ -0,0 +1,125 @@
+# Blockchain
+
```

Comment:
What is this section and what is its purpose? We need an intro.

Reply:
Adds a simple intro